usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/clients/MealManagerClient.java

```diff
@@ -34,36 +34,52 @@ public MealManagerClient() {
      * @param owner Username of the owner of the pet
      * @param petName Name of the pet
      * @param meal The meal entity that contains the attributes of the meal eaten by the pet
+     * @return The response code
+     * @throws ExecutionException When the retrieval fails
+     * @throws InterruptedException When the retrieval is interrupted
      */
-    public void createMeal(String owner, String petName, Meal meal) {
+    public int createMeal(String accessToken, String owner, String petName, Meal meal) throws ExecutionException,
+        InterruptedException {
         JSONObject reqJson = meal.getBody().buildMealJson();
         taskManager = taskManager.resetTaskManager();
         taskManager.setTaskId(POST);
         taskManager.setReqBody(reqJson);
-        taskManager.execute(BASE_URL + owner + "/" + petName + "/" + meal.getDate());
+        StringBuilder response = taskManager.execute(BASE_URL + owner + "/" + petName + "/" + meal.getDate(),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "/" appears 13 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5327591)